cmd/kola/options.go

```diff
@@ -35,6 +35,7 @@ var (
 	kolaDefaultImages  = map[string]string{
 		"amd64-usr": sdk.BuildRoot() + "/images/amd64-usr/latest/coreos_production_image.bin",
 		"arm64-usr": sdk.BuildRoot() + "/images/arm64-usr/latest/coreos_production_image.bin",
+		"s390x-usr": sdk.BuildRoot() + "/images/s390x-usr/latest/coreos_production_image.bin",
```

Comment:
You can drop this, CL doesn't produce s390x builds.

Reply:
for that matter we can drop arm too.

Reply:
There is a check https://github.com/coreos/mantle/blob/master/cmd/kola/options.go#L170 which fails the kola run, if there is no image for board - `Error: unsupport board "s390x-usr"`
I will add code to exclude s390x from the check. If you think it's necessary, I can add also arm to this list

Reply:
That code should probably be modified to check if the distribution is CL, it doesn't make sense in the context of RHCOS/FCOS. I'd say for now, leave the ARM entry and modify the `if` block like so:

```golang
if kola.Options.Distribution == "cl" && !ok {
    ...
}
```